app/javascript/src/javascripts/posts.js.erb

```diff
@@ -598,6 +602,27 @@ Post.initialize_saved_searches = function() {
   });
 }
 
+Post.update_tag_count = function(event) {
+  let string = "0 tags";
+  let count = 0;
+
+  if (event) {
+    let tags = $(event.target).val().match(/\S+/g);
+    if (tags) {
+      count = tags.length;
+      string = (count == 1) ? (count + " tag") : (count + " tags")
+    }
+  }
+  $("#tags-container .count").html(string);
+  if (count < 25) {
+    $("#tags-container .options #face").removeClass().addClass("far fa-frown");
+  } else if (count < 35) {
+    $("#tags-container .options #face").removeClass().addClass("far fa-meh");
+  } else {
+    $("#tags-container .options #face").removeClass().addClass("far fa-smile");
+  }
```

Comment:
Just a suggestion. Since we have `webpacker` and already using ES6+ syntax in some places, then we could write a little bit less code

```es6
let class_emoji = "smile";
if (count < 35) {
  class_emoji = count < 25 ? "frown" : "meh";
}
$("#tags-container .options #face").removeClass().addClass(`far fa-${class_emoji}`);
```